nepal_constitution_ai/data_pipeline/loader.py

```
@@ -1,12 +1,20 @@
+import os
+from loguru import logger
 from langchain_community.document_loaders import PyPDFLoader
 from nepal_constitution_ai.config.config import settings
-from loguru import logger
 
 def load_pdf():
+    file_path = settings.FILE_PATH
+    file_extension = os.path.splitext(file_path)[1].lower()
+
+    if file_extension != '.pdf':
+        logger.error(f"Invalid file type: {file_extension}. Expected a .pdf file.")
+        return None
```

I guess its better to raise exception here rather than returning None.